src/frontend/src/catalog/catalog_service.rs

```diff
@@ -105,6 +106,7 @@ impl CatalogWriter for CatalogWriterImpl {
                 id: 0,
                 name: schema_name.to_string(),
                 database_id: db_id,
+                owner: risingwave_common::catalog::DEFAULT_SUPPER_USER.to_string(),
```

Comment:
Should be same as database. Add an input param owner and use it instead of default user.